README.md

```diff
@@ -1,6 +1,202 @@
-# KubernetesAccessManager
-KubernetesAccessManager is a web application designed for multi-user management of Kubernetes cluster. It provides a graphical interface for users to manage their Kubernetes resources. Administrators can define precise access rights based on roles from an identity provider, such as Keycloak.
+<h1 style="display: flex; align-items: center; gap: 30px;">
+  Kubernetes Access Manager (KAM)
+  <img src="./docs/images/KAM_logo.png" alt="KAM Logo" style="height: 125px;">
+</h1>
```

Comment:
```suggestion
<h1 style="display: flex; flex-direction: column; align-items: center; gap: 30px;">
  <img src="./docs/images/KAM_logo.png" alt="KAM Logo" style="height: 125px;">
  Kubernetes Access Manager (KAM)
  <img src="https://github-readme-tech-stack.vercel.app/api/cards?title=Tech+Stack&align=center&lineCount=1&width=600&hideTitle=true&bg=%230D1117&badge=%23161B22&border=%2321262D&titleColor=%2358A6FF&line1=go%2Cgo%2C2596be%3Bkubernetes%2Ckuberenetes%2C2e6ce6%3Bhelm%2Chelm%2Cffffff%3Btypescript%2Ctypescript%2C2d79c7%3Breact%2Creact%2C00d8ff%3B" alt="Tech Stack" />
</h1>
```

![image](https://github.com/user-attachments/assets/f6085df3-6cb7-4a7b-b630-404f470d6220)

Reply:
![image](https://github.com/user-attachments/assets/68d10e85-5b0a-48e8-8ca9-3ea565e57df9)

Reply:
```suggestion
<h1 style="display: flex; align-items: center; gap: 30px;">
  Kubernetes Access Manager (KAM)
  <img src="./docs/images/KAM_logo.png" alt="KAM Logo" style="height: 125px;">
</h1>
```

![image](https://github.com/user-attachments/assets/ad3aa590-8b9d-41f3-9ffc-a356534a6f26)

Reply:
https://github.com/user-attachments/assets/bece63a6-583c-43be-a059-ad2c0d3fe2eb

Reply:
![time to make world war two look like a tea party](https://github.com/user-attachments/assets/46e067bb-9921-4af0-99f4-dc6376ff84eb)

Reply:
![ezgif-3-ebda669068-min](https://github.com/user-attachments/assets/dbc09936-cdc8-4064-9c2d-e1041a4adb11)
On my way to defend position of this logo

Reply:
![peace was never an option](https://github.com/user-attachments/assets/0a5a4f26-82c9-4ead-8e12-b411709ca3e0)

Reply:
![defeat-cait-sith.gif](https://github.com/user-attachments/assets/931cfaa1-f441-4c3f-a0a4-1606a2da9126)

Reply:
Dobra wczoraj w nocy przez przypadek mojego apply'a dałem xD